HedvigInsurance/web-onboarding
# GRW-858 Fix issues with editing bundles that include accident quotes

## What?

How to reproduce the bug
* Create a bundle that includes Accident
* Try to edit a field (say first name) from the edit details form
* Expected: Update will be successful, modal closes
* Actual: Modal closes, but nothing is updated

This also affects the checkout form
* Edit first name in checkout form
* Click sign: Update fails, but we still proceed to signing

## Root cause

We were not passing the "isStudent" param as part of "data" in the `quoteCart_createQuoteBundle` mutation. The causes the API to return an error we do not handle.

## Solution

* Add isStudent as part of Data
* Handle these types of errors when they happen

**Ticket(s): [GRW-858]**

## Screenshots / recordings 

https://user-images.githubusercontent.com/89857278/156156641-4777a9fd-b662-481c-bf48-b304be56a063.mov

### [Review app](https://web-onboardi-grw-858-fi-9bg0ob.herokuapp.com/se/new-member/offer-debugger)



[GRW-858]: https://hedvig.atlassian.net/browse/GRW-858?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
